apps/previewgenerator.sh

```diff
@@ -60,23 +74,32 @@ then
 #    occ_command config:system:set enabledPreviewProviders 20 --value="OC\\Preview\\Font"
     
     # Set aspect ratio
-    occ_command config:app:set --value="32 64 1024"  previewgenerator squareSizes
-    occ_command config:app:set --value="64 128 1024" previewgenerator widthSizes
-    occ_command config:app:set --value="64 256 1024" previewgenerator heightSizes
+    occ_command config:app:set previewgenerator squareSizes --value="32 256"
+    occ_command config:app:set previewgenerator widthSizes  --value="256 384"
+    occ_command config:app:set previewgenerator heightSizes --value="256"
+    occ_command config:system:set preview_max_x --value="2048"
+    occ_command config:system:set preview_max_y --value="2048"
+    occ_command config:system:set jpeg_quality --value="60"
+    occ_command config:app:set preview jpeg_quality --value="60"
     
     # Add crotab
-    crontab -u www-data -l | { cat; echo "@daily php -f $NCPATH/occ preview:pre-generate >> /var/log/previewgenerator.log"; } | crontab -u www-data -
+    crontab -u www-data -l | { cat; echo "0 4 * * * php -f $NCPATH/occ preview:pre-generate >> /var/log/previewgenerator.log"; } | crontab -u www-data -
     touch /var/log/previewgenerator.log
     chown www-data:www-data /var/log/previewgenerator.log
     
-    # Install needed dependencies
-    install_if_not ffmpeg
-    install_if_not libreoffice
-    install_if_not php-imagick
-    install_if_not libmagickcore-6.q16-3-extra
+    msg_box "In the last step you can choose to run the previewgenerator now. 
     
-    # Pre generate everything
-    occ_command preview:generate-all
+    Please note: If you choose to not doing it now, then you have to do it manually before the cronjob at the next day at 4 am can continue.
```

Comment:
Remove this, there shouldn't be an option *not* to run the pre-generate command.

Reply:
Done.